src/index.html

```diff
@@ -9,6 +9,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="logo">
+          <a href="#" class="logo-link">
+            <img href="logo" src="images/logo.png" alt="logo">
+          </a>
+        </div>
+        <nav class="for-nav">
+          <ul class="nav">
+            <li class="nav-item">
+              <a href="apple" class="nav-link is-active">APPLE</a>
+            </li>
+            <li class="nav-item">
+              <a href="sumsung" class="nav-link">SAMSUNG</a>
```

Comment:
Add achor to your links
```suggestion
              <a href="#sumsung" class="nav-link">SAMSUNG</a>
```